content/20.pharmaceuticals.md

```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
 This study found that several drugs had been identified as candidates across multiple independent analyses, including several that had received significant attention as potential therapeutics for COVID-19 (e.g., CQ/HCQ and remdesivir) [@doi:10.1016/j.patter.2020.100090].
-A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1101/2021.03.23.436648].
+A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1126/science.abi4708].
 Cationic amphiphilic drugs are known to initiate phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270].
 Mechanistically, phospholipidosis could disrupt viral replication by inhibiting lipid processing [@doi:10.1080/01913123.2020.1850966] (see discussion of HCQ in Appendix A).
 They found that antiviral activity against SARS-CoV-2 was correlated with phospholipidosis for drugs with and without cationic amphiphilicity.
 This observation is also consistent with preliminary evidence that certain psychotropic drugs may be protective against COVID-19 [@doi:10.1016/j.drudis.2020.06.022; @doi:10.1007/s00406-020-01231-x; @doi:10.1001/jama.2020.22760].
 While _in vivo_ application of this principle has not yet been successful, this effort demonstrates the potential for high-throughput screening to reveal general biological principles that may be beneficial in developing strategies to combat SARS-CoV-2.
 Other large-scale efforts to screen candidate drugs against SARS-CoV-2 (e.g., [@doi:10.1016/j.celrep.2021.108959]) may complement these efforts and provide additional insights into the fundamental characteristics of drugs that show potential for repurposing.
+<!-- I read the Tummino et al. study, and this isn't how I woudl characterize their findings. My understanding is that they hypothesize that these cationic amphiphilic drugs induce phospholipidosis, which has antiviral activity in in vitro assays but none in vivo. They are characterizing these compounds as confounders or false positives that were detected in the computational or experimental screens and define chemical structure-based rules to identify similar false positives. The current framing seems to imply that cationic amphiphilic drugs may still have clinical success, but the paper argues that pursuing these confounders (including CQ/HCQ) has been a big waste of time and money in clinical trials. I read the journal version, not the preprint, so the message may have changed. -->
```

Comment:
Thank you so much for catching this! I read it a while ago and I don't remember the details at all. Would the comments above address your concerns?

Reply:
I think so.  I'll commit all your changes and then read through the paragraph again.

Reply:
I made additional changes in 3dcd7e2a93d9c4bc0ab2078d5d664fa0325900b8 to match my interpretation of Tummino et al.  How does that look to you?